wallets/stytch/.editorconfig

```diff
@@ -0,0 +1,12 @@
+root = true
```

Comment:
Does this file need to be here? Or is it just for meeting patterns?

Reply:
Yea just meeting patterns. In order to add your own wallet they recommend just copying the "leap" wallet and making adjustments so this is a file that just got dragged along. Also the reason for the 31k lines